<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeurTCPIPClass</class>
 <widget class="QMainWindow" name="ServeurTCPIPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServeurTCPIP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="listenGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Serveur</string>
    </property>
    <widget class="QLabel" name="portLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="listenButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Mettre en Ã©coute</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="portLine">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="listenStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Etat serveur : HS</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="logBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="logLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logClearButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Effacer le log</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ServeurTCPIP.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listenButton</sender>
   <signal>clicked()</signal>
   <receiver>ServeurTCPIPClass</receiver>
   <slot>OnListenButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logClearButton</sender>
   <signal>clicked()</signal>
   <receiver>ServeurTCPIPClass</receiver>
   <slot>OnLogClearButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>newConnection()</signal>
  <slot>OnListenButtonClicked()</slot>
  <slot>OnServerNewConnection()</slot>
  <slot>OnLogClearButtonClicked()</slot>
 </slots>
</ui>
